GoMain/dockerfiles/Dockerfile_qemu_aarch64

```diff
@@ -0,0 +1,35 @@
+# Docker image for "edge-orchestration"
+# TODO - need to reduce base image of edge-orchestration
+### ubuntu:16.04 image size is 119MB
+### alpine:3.6 image size is 4MB
+ARG PLATFORM
+FROM $PLATFORM/ubuntu:16.04
+
+# environment variables
+ENV TARGET_DIR=/edge-orchestration
+ENV HTTP_PORT=56001
+ENV MDNS_PORT=5353
+ENV MNEDC_PORT=3334
+ENV MNEDC_BROADCAST_PORT=3333
+ENV ZEROCONF_PORT=42425
+ENV APP_MAIN_DIR=GoMain
+ENV APP_BIN_DIR=$APP_MAIN_DIR/bin
+ENV APP_NAME=edge-orchestration
+
+# copy files
+COPY $APP_BIN_DIR/$APP_NAME $APP_MAIN_DIR/run.sh $TARGET_DIR/
+COPY $APP_MAIN_DIR/qemu-aarch64-static /usr/bin/
```

Comment:
Assume that this file is the Dockerfile in the GoMain folder.
We can copy files related to qemu like below.
```suggestion
COPY $APP_QEMU_DIR/ /usr/bin/
```